onedal/cluster/kmeans.py

```diff
@@ -337,7 +338,7 @@ def _set_cluster_centers(self, cluster_centers):
     cluster_centers_ = property(_get_cluster_centers, _set_cluster_centers)
 
     def _predict_raw(self, X_table, module, policy, dtype=np.float32):
-        params = self._get_onedal_params(dtype)
+        params = self._get_onedal_params(dtype, "computeAssignments")
```

Comment:
specify param of the value, make it readable.
```suggestion
        params = self._get_onedal_params(dtype, result_options="computeAssignments")
```